core/templates/dev/head/pages/exploration-editor-page/services/learner-answer-details-data.service.ts

```diff
@@ -64,14 +64,14 @@ angular.module('oppia').factory('LearnerAnswerDetailsDataService', [
       },
       fetchLearnerAnswerInfoData: function() {
         return _fetchLearnerAnswerInfoData().then(function(response) {
-          learnerAnswerInfoData = angular.copy(
-            response.data.learner_answer_info_data);
+          learnerAnswerInfoData =
+            response.data.learner_answer_info_data;
```

Comment:
Does it not work if you do angular.copy? I think that is better.

Reply:
It works, I removed it because I didn't get the reason of copying the object, since it's not being changed by other logic, so it's ok to have the same reference. Does it make sense? If it isn't, I can put angular.copy back again.